src/main/java/staffconnect/model/person/Person.java

```diff
@@ -29,6 +30,8 @@ public class Person {
     private final Set<Tag> tags = new HashSet<>();
     private final Set<Availability> availabilities = new HashSet<>();
 
+    private Set<Meeting> meetings = new HashSet<>();
```

Comment:
This can't be ```final``` because of the way I assign a new set of ```meeting``` to a person. This is to allow for future expansions and ```meeting``` to be mutable and change regularly without creating redundant immutable copies

Reply:
But I agree that the rest of the fields like availabilities, name etc.. be remain as final because, they are not updated as frequently as ```meeting```

Reply:
I understand. Thanks for clarifying! 👍

Reply:
Correct me if im wrong, your implementation of `meeting` is intentionally different from availability/tags because you want to update the meeting attribute without creating a new person each time? 

imo, i think its not worth differing in implematation because `final `is usually used to protect the code (security and safety) over the time taken to create a new person.

This is debatable but for time concern, can be moved to a different iteration/pr for changes

Reply:
I understand that you require meeting to be mutable. However, I think there might have been a misunderstanding of `final`. `final` simply makes the object _reference_ unchangeable. The object it points to is not immutable by doing so, thus if `meetings` is set as `final`, it can never refer to another object, but the object it refers to may change state.